packages/wonder-blocks-popover/components/popover.js

```diff
@@ -172,11 +200,15 @@ export default class Popover extends React.Component<Props, State> {
     }
 
     renderPopper(uniqueId: string) {
-        const {placement} = this.props;
-        const {anchorElement} = this.state;
+        const {initialFocusId, placement} = this.props;
+        const {anchorElement, controlled} = this.state;
 
         return (
-            <FocusManager anchorElement={anchorElement}>
+            <FocusManager
+                anchorElement={anchorElement}
+                initialFocusEnabled={!controlled}
```

Comment:
Why do we only set the initial focus when the popover is controlled?

Reply:
It's to cover the case when the Popover Dialog is opened on page load, so created this to avoid stealing focus from another element (maybe another React Component that tries to set focus) inside the DOM.

Reply:
I was thinking that maybe `initialFocusEnabled` and `initialFocusId` could be merged by setting:
```
initialFocusId={!controlled ? initialFocusId : undefined}
```

Reply:
> So when the Popover is opened on page load what is the behavior and how does this differ from it being opened some time after the page is loaded?

That's a really good point, I'm thinking that maybe it would be good to steal focus no matter if the Popover is controller or uncontrolled. At the end, we are displaying the popover, so we should inform to the user to focus on this element.

And good call about merging both props, I'll update that :).